Conclusion (#22)
greenelab/knowledge-graph-review: content/05.conclusion.md

## Conclusion

> Knowledge graphs have demonstrated to be an advantageous resource for the biomedical domain.

Reviewer: ```suggestion
Knowledge graphs are becoming widely used in biomedicine, and we expect their use to continue to grow.
```

> These graphs are constructed from pre-existing databases populated by domain experts via manual curation along with automated systems that extract relationships from text.

Reviewer: ```suggestion
At the moment, most are constructed from databases derived from manual curation or from co-occurrences in text.
However, we expect that machine learning approaches will play a key role in quickly bringing new findings into these graphs.
```

> Typically knowledge graphs are utilized by representing them in a low dimensional space that is designed to capture the local and/or global structure of the graph.

Reviewer: ```suggestion
Representing these knowledge graphs in a low dimensional space that captures the local and global structure of the graph can enable many downstream machine learning analyses, and methods to capture this structure are an active area of research.
```

> Once constructed, this space is exploited by machine learning algorithms to perform biomedical tasks within the multi-omics, pharmaceutical, and clinical domains.

Reviewer: ```suggestion
```

> As the field progresses validation has a crucial role in advancing subsequent efforts.

Reviewer: ```suggestion
As with any field, rigorous evaluation that can identify key factors that drive success is critical to moving the field forward.
```

> Prevsiouly mentioned, validation remains a non-trivial task because experiments in this context demand a significant amount of time [@doi:10.1093/jamia/ocy117; @doi:10.1155/2017/2498957].

Reviewer: ```suggestion
For work with knowledge graphs, evaluation remains hard.
Experiments in this context require a significant amount of time and consequently resources [@doi:10.1093/jamia/ocy117; @doi:10.1155/2017/2498957].
```

> An open area of exploration is to develop approaches that can accelerate validation while improving upon the frequent blinded literature search.

Reviewer: ```suggestion
Moving from open ended and uncontrolled evaluations that consist of describing findings that are consistent with the literature to blinded evaluations of the literature support behind predictions and non-predictions would be a good first step.
```

> Plus, future methods will need to reassure that biases such as node degree bias doesn't cofound novel results [@url:https://greenelab.github.io/xswap-manuscript/].

Reviewer: ```suggestion
There are also well-documented biases related to node degree and degree distribution that must be considered for accurate evaluation [@url:https://greenelab.github.io/xswap-manuscript/].
The diversity of applications also hinder the development of a standardized set of expected evaluations.
```

> Following validation, we anticipate that future techniques will improve upon constructing low dimensional representations of knowledge graphs by incorporating data such as node and edge types.

Reviewer: ```suggestion
We anticipate that a fruitful avenue of research will be techniques that can produce low dimensional representations of knowledge graphs that distinguish between multple node and edge types.
```

> Plus, we foresee that future endeavors will have better insight on dealing with factors that can skew the strucutre of knowledge graphs such as experimental biases or missing data.

Reviewer: ```suggestion
There are also many different sources of bias that lead to spurious edges or incompleteness, and modeling these may support better representations.
```

> Nevertheless, knowledge graphs are an advantageous resources that have benefitted the biomedical domain.

Reviewer: ```suggestion
It is a promising time for research into the construction and application of knowledge graphs.
The peer reviewed literature is growing at an increasing rate and maintaining a complete understanding of even a specific area is becoming increasingly challenging for scientists.
One path that scientists can take to maintain a complete awareness of litterature is to become hyper-focused on specific parts of the graph to maintain near-complete knowledge of that area.
If advances in how these graphs are constructed, represented, and applied can enable the linking of fields, we may be able to enjoy the benefits of this detailed knowledge without losing the broader contextual links.
```

## Conclusion

> 

Reviewer: It feels like many of the things in this section are restatements of what you've already said in the rest of the manuscript. Instead, could you write what the implications are?

I'm imagining it would start with:
* Knowledge graphs are being widely applied.
* Machine learning methods for automated extraction from text can make these more complete.
* Representing things in a low-dimensional space make these practical and new methods are being developed to best capture structure.

Then discuss in another paragraph:
* Evaluation is important to advances that move the field forward.
* Approaches to evaluation that can consider network biases will be important (cite xswap from @zietzm ).
* Better deployment of blinded literature review to avoid uncontrolled validation-by-google.

And perhaps finally close with:
* Likely important to capture networks with many node/edge types in the same space in a way that retains this complexity.
* Likely important to take a more sophisticated view of missingness as many different processes drive the gaps. (@jjc2718 has really crystalized this in my mind recently)

I think this has a lot of what you're saying here but it frames it differently than it is framed elsewhere in the paper. What's here right now is more of a restating in a very similar way to how you already covered it.
